### ISSUE TITLE
Donation Approved & Scheduled Email - Volunteer

### ISSUE THREAD

**Issue by scallait** [hack4impact-calpoly/habitat-for-humanity#174]
<img width="305" alt="image" src="https://user-images.githubusercontent.com/71291167/223966409-6e9118e8-a3e2-41a0-8c7f-2fbeba5e821d.png">

An email notification should be sent to the volunteer upon the admin approving a donation for pick up. Should occur in the final page of the donation approval and scheduling process shown above

Information that should be included in the email:
1. Volunteer first name
2. Donor first name, last name, phone number
3. Scheduled pick up time/date and Donor pick up address
4. Digital Receipt
5.  Habitat for Humanity Phone #, Email, Location, hours, website

### Prerequisites:
- #111 
- #171 
- #173 

### Required Specs:
- [ ] All information listed above is included in email

### Nice to Have Specs:
- None

#### Recommendations
-

**Comment by lukewaltz** [comment]
Blocked by #173

**Comment by ljbarker** [comment]
need to update to use sendgrid instead of nodemailer
